pyproject.toml

```
@@ -36,7 +36,7 @@ dependencies = [
   "rich ~= 13.0",
   "securesystemslib",
   "sigstore-protobuf-specs ~= 0.2.2",
-  "sigstore-rekor-types >= 0.0.11",
+  "sigstore-rekor-types >= 0.0.12",
```

Let's do an exact match, to avoid any pain in the future here :slightly_smiling_face: 

```suggestion
  "sigstore-rekor-types == 0.0.12",
```